### PR TITLE
AT_01.08.13 Header Log out button

### PR DESCRIPTION
**Pull request by IrinaLeibutina**
https://trello.com/c/pwbR66Qf/736-at0108013-verify-header-log-out-button

### CODE REVIEW

File: cypress/e2e/headerLogOutButton.cy.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/// <reference types="cypress"/>
+
+describe('Header | Log Out Button', () => {
+    
+    it('Log Out button is visible', () => {
+        cy.visit(`http://localhost:8080/`)
+        cy.get('#page-header > div.login.page-header__hyperlinks > a:nth-child(4)')
```
Comment (DariaMas7) [inline cypress/e2e/headerLogOutButton.cy.js:7]: If you want to check the visibility of the Log out button you should put some assert in the end

**Review by DariaMas7 — COMMENTED** [review]
In overall we use only 1 it() inside of describe and this 2 it() might be joined in 1

**Comment by DariaMas7** [comment]
Also in the description, you should attach the link to your TC from Trello

**Comment by SvetlanaVet** [comment]
Please, put on a link for the automation test case in the description

File: cypress/e2e/headerLogOutButton.cy.js (hunk 1 above)
Comment (SvetlanaVet) [reply cypress/e2e/headerLogOutButton.cy.js:7]: The port is better not to hardcode. Look at [#143](https://github.com/RedRoverSchool/JenkinsQA_JS_06/pull/143)

**Comment by IrinaLeibutina** [comment]
Remarks corrected

**Review by matvlesia — APPROVED** [review]
I think you can avoid using 'contains' by using this selector instead a[href='/logout']

**Review by ElenaKanash — APPROVED** [review]
LGTM

**Pull request merged**